File: python/prograimg_python_basic/Exercises/conditional_statements/conditional_statements_advanced_-_exercise/journey.py
budget = float(input())
session = str(input())

country = ""
spent = 0
place = ""

if budget <= 100:
    country = "Bulgaria"
    if session == "summer":
        spent = 0.30
        place = "Camp"
    elif session == "winter":
        spent = 0.70
        place = "Hotel"

elif budget <= 1000:
    country = "Balkans"
    if session == "summer":
        spent = 0.40
        place = "Camp"
    elif session == "winter":
        spent = 0.80
        place = "Hotel"

else:
    country = "Europe"
    spent = 0.90
    place = "Hotel"

spent_money = budget * spent

print(f"Somewhere in {country}")
print(f"{place} - {spent_money:.2f}")

File: JS/JavaScript/JS-Basics/ConditionalStatementsAdvancedExercise/Journey.js
function vacation(inputs) {
    
    let budget = parseFloat(inputs[0]);
    let session = String(inputs[1]);

    let destination;
    let amount;
    let typeVacation;

    if (budget <= 100) {
        destination = "Bulgaria";
        if (session == "summer") {
            amount = budget * 0.30;
            typeVacation = "Camp";
        }
        else if (session == "winter") {
            amount = budget * 0.70;
            typeVacation = "Hotel";
        };
    }   
    else if (budget <= 1000) {
        destination = "Balkans";
        if (session == "summer") {
            amount = budget * 0.40;
            typeVacation = "Camp";

        }
        else if (session == "winter") {
            amount = budget * 0.80;
            typeVacation = "Hotel";

        };
    }
    else if (budget > 1000) {
        destination = "Europe";
        amount = budget * 0.90
        typeVacation = "Hotel";
    };

    console.log(`Somewhere in ${destination}`)
    console.log(`${typeVacation} - ${amount.toFixed(2)}`)

}

vacation(["50", "summer"])
vacation(["75", "winter"])
vacation(["312", "summer"])
vacation(["678.53", "winter"])
vacation(["1500", "summer"])

